packages/collector/test/tracing/cloud/azure/blob/test.js

```diff
@@ -0,0 +1,477 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const { v4: uuid } = require('uuid');
+const expect = require('chai').expect;
+const semver = require('semver');
+const constants = require('@instana/core').tracing.constants;
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
+const config = require('../../../../../../core/test/config');
+const { verifyHttpRootEntry, expectExactlyNMatching } = require('../../../../../../core/test/test_util');
+const ProcessControls = require('../../../../test_util/ProcessControls');
+const globalAgent = require('../../../../globalAgent');
+const { fail } = expect;
+const testUtils = require('../../../../../../core/test/test_util');
+const { createContainer, deleteContainer, minimumNodeJsVer } = require('./util');
+const { BlobServiceClient } = require('@azure/storage-blob');
+const containerName = `nodejs-team-${uuid()}`;
+const storageAccount = process.env.BLOB_STORAGE_ACCOUNT;
+const accountKey = process.env.BLOB_ACCOUNT_KEY;
+
+/**
+ * endPoint is assigned with the default constant string for appending EndpointSuffix
+ * to connection string of an azure account
+ */
+const endPoint = 'EndpointSuffix=core.windows.net';
+const connStr = `DefaultEndpointsProtocol=https;AccountName=${storageAccount};AccountKey=${accountKey};${endPoint}`;
+const blobServiceClient = BlobServiceClient.fromConnectionString(connStr);
+const containerClient = blobServiceClient.getContainerClient(containerName);
+
+let mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
+mochaSuiteFn = semver.lt(process.versions.node, minimumNodeJsVer) ? describe.skip : mochaSuiteFn;
+
+/**
+ * This suite is skipped if no storageAccount or accountKey has been provided via BLOB_STORAGE_ACCOUNT
+ * and BLOB_ACCOUNT_KEY. For the Azure blob tests, the azure storage account used is teamnodejstracer
+ * which is the value for BLOB_STORAGE_ACCOUNT. From the azure portal, navigate to this storage account
+ * and under the Access keys, Key can be found for BLOB_ACCOUNT_KEY.
+ */
+if (!storageAccount || !accountKey) {
+  describe('tracing/cloud/azure/blob', function () {
+    it('The configuration for Azure is missing', () => {
+      fail('Please set process.env.BLOB_ACCOUNT_KEY and process.env.BLOB_STORAGE_ACCOUNT before tests.');
+    });
+  });
+} else {
+  mochaSuiteFn('tracing/cloud/azure/blob', function () {
+    globalAgent.setUpCleanUpHooks();
+    const agentControls = globalAgent.instance;
+
+    const controls = new ProcessControls({
+      dirname: __dirname,
+      useGlobalAgent: true,
+      env: {
+        CONTAINER_NAME: containerName,
```

Comment:
Could you please add a prefix such as `AZURE_CONTAINER_NAME`?
See [example](https://github.com/instana/nodejs/blob/v3.0.0/packages/collector/test/tracing/database/couchbase/test.js#L106).

Reply:
fixed